projects/demo/src/modules/app/home/home.component.ts

```diff
@@ -3,7 +3,7 @@ import {changeDetection} from '@demo/emulate/change-detection';
 import {encapsulation} from '@demo/emulate/encapsulation';
 
 @Component({
-    selector: `home`,
+    selector: `demo-home`,
```

Comment:
Selector of a component that uses ViewEncapsulation.ShadowDom must contain a hyphen.